tests/foreman/cli/test_host.py

```diff
@@ -2324,6 +2324,58 @@ def test_syspurpose_end_to_end(
         )
 
 
+@pytest.mark.rhel_ver_match('[^7]')
+def test_multi_cv_reg_not_allowed(
```

Comment:
Lets also change this to a test_positive_multi_cv_reg_not_allowed or a test_negative_multi_cv_reg_not_allowed. In my mind... this is a test negative scenario